letter.md

```diff
@@ -1,25 +1,21 @@
+Because of its success, CRAN has become a fundamental global resource, not only for independent researchers, but also for large organizations across academia, government, and industry, all of whom all depend on the ability to contribute packages in a reliable and predictable manner. The scope and the need have grown bigger than one individual or small team, and it is appropriate that the R community exercise a voice with respect to the conduct and administration of CRAN.
 
-Unfortunately, we, the undersigned R community members, are unified in our concern that CRAN is showing signs that it is struggling to create a consistent experience for contributors. 
+We, the undersigned R community members, are unified in concern that CRAN is struggling to provide a consistent experience for contributors.
 
-In particular there appears to be pattern of increasing and shifting the burden of maintaining packages on CRAN via regular introduction of new criteria and policies, coupled with an uptick in heavy-handed enforcement practices. Some of these new constraints are not well documented, causing much confusion that is adding to the burden on both contributors and CRAN administrators.
+New criteria and policies are regularly introduced, sometimes without enough documentation for contributors to follow, and enforcement practices have become increasingly discourteous and severe. Contributors are experiencing ambiguity, disruption, and frustration, and they are changing their behaviour to reduce exposure to CRAN's policies and checks (e.g. skipping all tests on CRAN). This not only undermines CRAN's checking system but also the R language itself since CRAN checks are used to validate core changes.
 
-This pattern is driving behavioural change in contributors among us. Many are acting to reduce their exposure to CRAN's policies and checks. This not only undermines CRAN's checking system but also the R language itself since CRAN checks are used to validate core changes.
+To reverse these trends, we are petitioning CRAN for reform. We request measures that:
 
-There are also contributors for whom the increasing burden has reached a tipping point, leading them to feel frustrated and discouraged. There is evidence that this frustration is mirrored by some CRAN administrators with numerous accounts of communications that are apparently coloured by it. 
+* Establish channels for CRAN collaboration with the R community.
+* Clarify criteria for packages to be accepted on CRAN.
+* Ensure a predictable experience for contributors.
+* Ensure CRAN is accessible to a diverse set of contributors.
 
-To reverse these trends we believe it is time for CRAN to undertake reform. At a high level we are calling for measures that will:
+Communication between the community and CRAN precedes our other proposed reforms, and we request an avenue by which R community members can directly engage CRAN administrators on the topics described here.
```

Comment:
This was part of my attempt to paraphrase the original writing of the letter. I have not personally attempted to engage CRAN, I think you and Miles would know better than I about the specifics of the current communication situation.

Reply:
I haven't engaged much with CRAN besides submitting 2 packages (I refused to change some links which then required to change links redirections on Bioconductor and attempted to get feedback from an analysis of CRAN review system which I didn't get any comment or reply) or R core members (I have indireclty requested to update bugzilla, and change the mailing list to avoid storing the passwords in plain text, I've only directly interacted with them, a subset, about Bugzilla and my package bugRzilla and on R contributor working meeting), but I think this is the most unclear point of this rewrite.

Reply:
There isn't much more about CRAN communication that I know of, you either send an email to the generic site, or send it to some R core members working on CRAN (besides them there are two people from Austria reviewing packages, which is a bit unclear why but I think this could be used to point out that other people can also review on CRAN without being R core members). 

There isn't a venue to ask CRAN for a position. I recently tried about taking over a package and the response was to give more details on a private email to CRAN (instead of sharing a position and arguments of how to make the transition, or simply saying that we need to discuss this wait until we get back to the R-pkg-devel mailing list).

Reply:
I can see this is a kind of compression of what was here before, but it's a little too brief to be clear to me. Though, it has made it clear to me that what was there before wasn't the smoothest.

I am also hesitant to drop the point that CRAN will need to improve its communication to attract more resources. It pairs well with the assertions about scale that have been added above.

Reply:
I added back some of the original wording and expanded that paragraph a bit.